todo/src/reducers/reducers.js

```diff
@@ -0,0 +1,41 @@
+
+const initialState = {
+    todos: [ ]
+}
+
+const reducer = (state = initialState, action) => {
+    switch(action.type){
+        case "ADD_ITEM":
+            return Object.assign( 
```

Comment:
You don't need to use both Object.assign and the spread operator. You could just return `todos: [...this.state.todos, {id: action.id, value: action.value, completed:false }]`